js/PixelPainter.js

```diff
@@ -0,0 +1,125 @@
+window.PixelPainter = (function(){
+
+  let paint = {};
+
+  const doc = document;
+  const body = document.querySelector('body');
+  let currentColor = "color";
+  let e = event;
```

Comment:
All the locations that us `e` is used it is defined by the argument being passed to a function. This assignment is not really being used.